projects/plugins/boost/app/lib/critical-css/Critical_CSS_State.php

```diff
@@ -386,6 +386,14 @@ public function is_fatal_error() {
 		return self::FAIL === $this->state;
 	}
 
+	public static function is_fresh() {
+		return Transient::get( 'ccss_is_fresh', false );
+	}
+
+	public static function set_fresh( $is_fresh = true ) {
+		Transient::set( 'ccss_is_fresh', $is_fresh, WEEK_IN_SECONDS );
+	}
```

Comment:
🤔 Is there a reason to use this instead of the age of the stored Critical CSS on file? We currently show "generated X hours ago" on the page - can we use the same info to determine if the css is fresh?

Reply:
We can, but we will not have the option to dismiss the dialog for a week temporarily.